pkg/http/client_test.go

```diff
@@ -3,14 +3,15 @@ package http_test
 import (
 	"context"
 	"fmt"
-	cfgMocks "github.com/applike/gosoline/pkg/cfg/mocks"
-	"github.com/applike/gosoline/pkg/http"
-	monMocks "github.com/applike/gosoline/pkg/mon/mocks"
-	"github.com/stretchr/testify/assert"
 	netHttp "net/http"
 	"net/http/httptest"
 	"testing"
 	"time"
+
+	cfgMocks "github.com/applike/gosoline/pkg/cfg/mocks"
```

Comment:
please adjust your gofmt configuration, so it produces the same output as it originally was. Or use goland and activate gofmt in the file watchers.